config/locales/server.en.yml

```diff
@@ -3458,6 +3458,7 @@ en:
   subject_re: "Re: "
   subject_pm: "[PM] "
   email_from: "%{user_name} via %{site_name}"
```

Comment:
Is it safe to remove `:email_from` or something else is still using it??

Reply:
We still use it for regular non-group-smtp emails so we have to keep it.